packages/at_client/lib/src/service/notification_service_impl.dart

```diff
@@ -319,6 +319,7 @@ class NotificationServiceImpl
           true, // this was the behaviour before introducing this parameter
       bool checkForFinalDeliveryStatus =
           true, // this was the behaviour before introducing this parameter
+      bool encryptValue = true,
```

Comment:
please add the comment `// this was the behaviour before introducing this parameter`

Reply:
Done